pfsspy/__init__.py

```diff
@@ -512,6 +513,74 @@ def _common_b(self):
         self._common_b_cache = br, bs, bp, Sbr, Sbs, Sbp
         return self._common_b_cache
 
+    def get_Bvec(self,skycoord,out_type="cartesian") :
+        """
+        Input
+        =====
+        skycoord : astropy.SkyCoord 
+        An arbitary point or set of points (length N >= 1) 
+        in the PFSS model domain (1Rs < r < Rss)
+
+        Output
+        ======
+        bvec : ndarray (ndarray.shape = (N,3), units nT) 
+        Magnetic field vectors at the requested locations 
+
+        Keyword Arguments
+        =================
+        coords_type : str
+        Either "cartesian" or "spherical" depending on desired
+        output type :
+            Cartesian -> Bx,By,Bz. 
+            Spherical -> Br, B_theta, B_phi 
+            (physics convention : https://upload.wikimedia.org/wikipedia/commons/thumb/4/4f/3D_Spherical.svg/240px-3D_Spherical.svg.png)
```

Comment:
I'm tempted to keep this simple, and just output spherical coordinates, since the solver is all in spherical coordinates. That would mean avoiding having to deal with coordinate conversion conventions etc. for the cartesian case. Thoughts?

Reply:
I agree this would be simpler. The reason it's there was the specific application that spurred me to write this PR involved tracing the vectors a long a line of sight through the PFSS model, for which cartesian vectors were much more useful. The transformation is pretty straightforward and could be left to the user. That said, as long as it's a keyword agument with spherical as the default, I think it's ok to have the option - what do you think?

Reply:
I think the default should definitely be spherical. If there's going to be a cartesian option I think that's fine, but there needs to be a full (mathematical) description of the transformation between the two in the pfsspy docs somewhere.

Reply:
see latest commit, I made spherical the default (I'll just note here that the internal interpolator brgi does output cartesian coordinates), and I added details of the transformation in the docstring